File: coding2025/Programers/level1/Java/크기가작은부분문자열.java
package coding2025.Programers.level1.Java;

public class 크기가작은부분문자열 {
    public int solution(String t, String p) {

        int len = p.length();
        long num = Long.parseLong(p);
        int ans = 0;

        for(int i=0; i<t.length()-len+1; i++){
            long diff = Long.parseLong(t.substring(i, i+len));
            if(diff<=num) ans++;
        }
        return ans;
    }
}

File: AlgoPython/lev1/크기가작은부분문자열.py
def solution(t, p):


    ans = 0
    p_len = len(p)
    t_len = len(t)
    p = int(p)

    for i in range(0, t_len+1-p_len):
        if int(t[i:i+p_len]) <=p:
            ans +=1

    return ans